### PR TITLE
Stylize header

### PR DESCRIPTION
**Pull request by curran**
This PR makes the title part of the page look like the design in #9 
image

This also includes a change where multiple font weights are loaded.

### CODE REVIEW

**Comment by curran** [comment]
@vidonne Couldyou please review and merge if this looks good? Thanks!

File: index.html [hunk 1]
```diff
@@ -9,15 +9,16 @@
...
 
     <link href="styles.css" rel="stylesheet">
 
   </head>
   <body>
     <img class="unhcr-logo-small" src="images/unhcr-logo-horizontal.svg">
-    <div class="unhcr-headline">
-      UNHCR Data Visualization Style Guide
+    <div class="unhcr-page-title">
+      <strong>DATAVIZ</strong>
```
Comment (vidonne) [inline index.html]: I changed it to "Graphics Styleguide" as we are covering more than dataviz I guess. Nothing major

File: styles.css [hunk 2]
```diff
@@ -8,3 +8,19 @@
   width: 200px;
   display: inline-block;
 }
+
+.unhcr-page-title {
+  display: inline-block;
+  margin-left: 10px;
```
Comment (vidonne) [inline styles.css]: margin-left: 16px;

File: styles.css [hunk 3]
```diff
@@ -8,3 +8,19 @@
...
+  font-family: 'Proxima Nova', 'Lato', Arial, sans-serif;
+  font-size: 38pt;
```
Comment (vidonne) [inline styles.css]: font-size: 36px;

File: styles.css [hunk 4]
```diff
@@ -8,3 +8,19 @@
...
+}
+
+.unhcr-page-title strong {
+  font-weight: 900;
```
Comment (vidonne) [inline styles.css]: font-weight: 700;

File: styles.css [hunk 5]
```diff
@@ -8,3 +8,19 @@
   width: 200px;
```
Comment (vidonne) [inline styles.css]: if this affect the logo on unhct.org it is actually based on the height: 48px;

File: index.html [hunk 6]
```diff
@@ -9,15 +9,16 @@
       If you have a license for the Proxima Nova font, you can
       include it here and comment out the Lato link tag below.
     -->
-    <link href="https://fonts.googleapis.com/css?family=Lato" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700,900" rel="stylesheet">
```
Comment (vidonne) [inline index.html]: Probably won't need the weight:900

File: styles.css (hunk 4 above)
Comment (curran) [inline styles.css]: Aha! Thanks for this. I couldn't tell based on the mockup.

File: styles.css (hunk 5 above)
Comment (curran) [inline styles.css]: Great to know!

**Comment by curran** [comment]
This is how it looks now, after those changes:

image

The 36pt causes the text not align with the top of the logo.

By comparison, here's what 38 looks like:

image

and here's what 40 looks like:

image

**Comment by curran** [comment]
@vidonne Ready for a second review.

**Comment by vidonne** [comment]
It's funny on my side it looks like this with 36px 
dataviz-styleguide-website_ai____100___rgb_gpu_preview_

**Comment by curran** [comment]
It looks like that when you load the page locally? In which browser?

**Comment by curran** [comment]
Merging, as this is an improvement. We can always tweak it more later.

**Pull request merged**